tests/ImageSharp.Benchmarks/Codecs/Jpeg/EncodeJpeg.cs

```diff
@@ -12,23 +13,44 @@ namespace SixLabors.ImageSharp.Benchmarks.Codecs.Jpeg
 {
     public class EncodeJpeg
     {
-        // System.Drawing needs this.
-        private Stream bmpStream;
+        [Params(50, 75, 95, 100)]
+        public int Quality;
+
+        private const string TestImage = TestImages.Jpeg.BenchmarkSuite.Jpeg420Exif_MidSizeYCbCr;
+
+        // GDI+ uses 4:2:0 subsampling
+        private const JpegSubsample EncodingSubsampling = JpegSubsample.Ratio420;
+
+        // System.Drawing
         private SDImage bmpDrawing;
+        private Stream bmpStream;
+        private ImageCodecInfo jpegCodec;
+        private EncoderParameters encoderParameters;
+
+        // ImageSharp
         private Image<Rgba32> bmpCore;
+        private JpegEncoder encoder;
+
         private MemoryStream destinationStream;
 
         [GlobalSetup]
         public void ReadImages()
         {
             if (this.bmpStream == null)
             {
-                const string TestImage = TestImages.Jpeg.BenchmarkSuite.Jpeg420Exif_MidSizeYCbCr;
                 this.bmpStream = File.OpenRead(Path.Combine(TestEnvironment.InputImagesDirectoryFullPath, TestImage));
+
                 this.bmpCore = Image.Load<Rgba32>(this.bmpStream);
                 this.bmpCore.Metadata.ExifProfile = null;
+                this.encoder = new JpegEncoder { Quality = Quality, Subsample = EncodingSubsampling };
+
                 this.bmpStream.Position = 0;
                 this.bmpDrawing = SDImage.FromStream(this.bmpStream);
+                this.jpegCodec = GetEncoder(ImageFormat.Jpeg);
+                this.encoderParameters = new EncoderParameters(1);
```

Comment:
Note. `EncoderParameters` implement `IDisposable` but the GC should clean up eventually here.

Reply:
Shouldn't be hard to fix. And I completely forgot to update benchmarks in the comments of that file. Plus It's better to implement full set of benchmarks for: both subsampling methods (skip 444 for System.Drawing) and a set of quality parameters, something like 75, 90, 100 for a full picture.

Reply:
If you have time that would be great!

Reply:
Done!